api/registrations/serializers.py

```diff
@@ -314,8 +314,13 @@ class RegistrationFileSerializer(FileSerializer):
     comments = FileCommentRelationshipField(related_view='registrations:registration-comments',
                                             related_view_kwargs={'node_id': '<node._id>'},
                                             related_meta={'unread': 'get_unread_comments_count'},
-                                            filter={'target': 'get_file_guid'})
+                                            filter={'target': 'get_file_guid'}
+                                            )
 
+    node = RelationshipField(related_view='registrations:registration-detail',
```

Comment:
I checked; it's fine. That was mostly a note in case anyone happened to think of the same case on a subsequent review.